x-pack/test/functional/page_objects/monitoring_page.js

```diff
@@ -23,19 +22,9 @@ export function MonitoringPageProvider({ getPageObjects, getService }) {
     }
 
     async assertTableNoData(subj) {
-      await retry.try(async () => {
-        if (!await testSubjects.exists(subj)) {
-          throw new Error('Expected to find the no data message');
-        }
-      });
-    }
-
-    async assertEuiTableNoData(subj) {
```

Comment:
Isn't `assertEuiTableNoData` still used in other places:
![Screen Shot 2019-10-07 at 7 55 23 PM](https://user-images.githubusercontent.com/27822304/66357388-9fcf8e00-e93c-11e9-9a14-03583946b3a2.png)

Reply:
Yes indeed, but [this change](https://github.com/elastic/kibana/pull/47486/files#diff-360d71f4898cc9d5d295451ac09397dbR62) will affect those tables too so it should be consistent

Reply:
@chrisronline Sorry, I'm not really understanding the change here. Looks like you changed `assertEuiTableNoData` to `assertTableNoData` in the `cluster_list.js` file in this PR. Shouldn't you do the same in:
- `elasticsearch_indices.js`
- `elasticsearch_nodes.js`
- `logstash_pipelines.js`

Otherwise, it'll try to reference a function that does not exist, right?

Reply:
Yes, very good point. I overlooked that. Thanks!